tests/cases/conformance/es6/computedProperties/computedPropertyNamesContextualType7_ES5.ts

```diff
@@ -6,9 +6,9 @@ interface I<T> {
 declare function foo<T>(obj: I<T>): T
 
 foo({
-    p: "",

```

Comment:
I'll add a test of `p` with a string indexer, although I'm still not sure what this test is even asserting here. From the name it sounds like it tests that properties with computed names get a contextual type. In which case I don't know what assertion it's making about a non-computed property.

Here's the new test I'll add:

```ts
interface J<T> {
  [s: string]: T
}
declare function g<T>(obj: J<T>): T;
g({ p: "" });